src/areaCalculator/AreaCalculator.java

```diff
@@ -1,11 +1,43 @@
 package areaCalculator;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class AreaCalculator {
 
-    public double calculateAreaOfRectangle(Rectangle rectangle) {
-        double area = rectangle.getLength() * rectangle.getWidth();
-        System.out.println("Area of rectangle (" + rectangle.getLength() + " x "
-                + rectangle.getWidth() + ") = " + area);
-        return area;
+    private List<Shape> shapeList;
+
+    public AreaCalculator() {
+        shapeList = new ArrayList<>();
+    }
+
+    void addShapeToList(Shape shape) {
+        shapeList.add(shape);
+    }
+
+    public void calculateArea() {
+        for (Shape shape : shapeList) {
+            if (shape instanceof Rectangle) {
+                calculateRectangleArea(shape);
+            }
+            if (shape instanceof Circle) {
```

Comment:
I think you want  else if, because  this if now  will be checked even if the shape was rectangle